docs/faq.md

```diff
@@ -0,0 +1,38 @@
+# FAQs
+Here is a list of some frequently asked questions about **#matrix**.
+
+- [How do I ask for help?](#How-do-I-ask-for-help?)
+- [What if I find a bug?](#What-if-I-find-a-bug?)
+- [Can I request a new feature?](#Can-I-request-a-new-feature?)
+- [Does #matrix do X?](#Does-#matrix-do-X?)
+- [How do I integrate #matrix and our single-sign on (SSO) solution?](#How-do-I-integrate-#matrix-and-our-single-sign-on-(SSO)-solution?)
+
+
+## How do I ask for help?
+You can use Stackoverflow or Slack for that. Please, do not open a support ticket in any [Resultados Digitais channel](https://resultadosdigitais.com.br/).
+
+* https://stackoverflow.com/questions/tagged/hash-matrix
+* http://hash-matrix.slack.com/ ([Invite here](https://join.slack.com/t/hash-matrix/shared_invite/zt-cwglw6te-kMlJiimq7qn4WeSTiv91og))
+
+## What if I find a bug?
+If you come across something that looks like a bug, start by searching our [Github issues](https://github.com/ResultadosDigitais/matrix/issues) to see if it has already been reported. If it has, please let us know you're experiencing the same issue by reacting with a thumbs up emoji or adding a comment providing additional information.
+
+If the bug has not yet been reported, go ahead and [open a bug report](https://github.com/ResultadosDigitais/matrix/issues/new/choose). We suggest collecting the following information to help us reproduce the issue:
+
+1. Server logs
+2. Javascript console logs
+3. Can it be reproduced on the sample dataset? (Rooms / Enviroments config)
+4. Your #matrix version
+5. Where #matrix is running (Docker image, AWS Elastic Beanstalk, Heroku, Linux/Ubuntu, etc)
+6. What browser version
+7. Screenshots (if relevant)
+
+## Can I request a new feature?
+Absolutely! New features can be added as issues in our [Github repo](ttps://github.com/ResultadosDigitais/matrix/issues). Before filing, take a minute to search and see if your feature has already been requested. If it has, it’s better to leave a thumbs up reaction on the existing issue than to file a new one. We use the thumbs up to measure how interested our community is in new features, so it’s better to have all of the thumbs ups captured on one issue.
+
+
+## Does #matrix do X?
+If you’re not sure if #matrix has the capability you’re looking for, we would recommend asking in [Slack](http://hash-matrix.slack.com/). If it’s something we don’t do, our friendly community may be able to point you towards a workaround, or the Github issue where the feature has been requested.
```

Comment:
We could put a badge at the beginning to indicate Slack, many people will not even get here at the end, and it also avoids an extra topic. What do you think?